Horatio/JSON/JSONParsing.swift

```diff
@@ -32,17 +32,23 @@ public struct JSONParsingOptions: OptionSet {
  types and missing values.
 */
 open class JSONParser {
-    open static func parseIdentifier(_ value: AnyObject?, options: JSONParsingOptions = .none) -> String? {
-        if let stringValue = JSONParser.parseString(value) {
+    open static func parseIdentifier(_ value: AnyObject?, ignoreDefault defaultToIgnore: String? = nil, options: JSONParsingOptions = .none) -> String? {
```

Comment:
This should be done as an extension to JSONParser that's app-specific; it's a bit of a degenerate case, so not something I'd want to put in the baseline.